e2e/bitrise.yml

```diff
@@ -132,24 +132,6 @@ workflows:
         inputs:
         - gradle_task: assembleDebug
 
-  test_support_library:
-    envs:
-    - SAMPLE_APP_URL: https://github.com/bitrise-io/sample-apps-android-x.git
-    - BRANCH: master
-    - GRADLE_BUILD_FILE_PATH: build.gradle
-    - GRADLEW_PATH: ./gradlew
-    before_run:
-    - _setup
-    steps:
-    - script:
-        title: Uninstall Android SDK
-        inputs:
-        - content: |-
-            #!/usr/bin/env bash
-            sdkmanager --uninstall "extras;android;m2repository"
```

Comment:
This was only a problem in very old Android Gradle Plugin versions. This even predates the support library vs. AndroidX migration thing. I don't think it's needed anymore, and it would be almost impossible to set up a working project that tests this successfully (this needs an old AGP version, which needs an old Gradle version, which needs JDK8, etc.)